snippets/social-meta-tags.liquid

```diff
@@ -84,7 +83,6 @@
   <meta name="twitter:data2" content="In stock">
   {% endif %}
 {% elsif template contains 'article' %}
-  <meta name="twitter:card" content="summary">
   <meta name="twitter:title" content="{{ article.title }}">
   <meta name="twitter:description" content="{{ article.excerpt_or_content | strip_html | truncatewords: 140, '' | escape }}">
   {% comment %}
```

Comment:
@carolineschnapp "at times like these don't you wish we had article.image"... just about every day!